index.html

```diff
@@ -3,9 +3,20 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <title>CV</title>
 </head>
 <body>
-    <h1>Hello world!</h1>
+    <header class="showcase">
+        <div class="main">
```

Comment:
probably it would be better to call it ` hero-section` instead of `main`

Reply:
Ok, I will change it tommorow.